packages/component-library/src/theme.scss

```diff
@@ -30,6 +30,21 @@ $font-weight: (
   @return map-get-strict($font-weight, $weight);
 }
 
+$font-family: (
+  "monospace": (
+    "ui-monospace",
+    "sfmono-regular",
+    "consolas",
+    "Liberation Mono",
+    "menlo",
+    "monospace",
+  ),
+);
+
+@function font-family($family) {
+  @return map-get-strict($font-family, $family);
+}
```

Comment:
I think this is great but at some point we should probably have a specific font we ship for mono text, cc @partyparrotgreg not sure if you already have one in mind